openfeedback-m3/src/main/kotlin/io/openfeedback/android/m3/OpenFeedbackLayout.kt

```diff
@@ -54,10 +54,24 @@ fun OpenFeedback(
                 sessionFeedback = session,
                 modifier = modifier,
                 columnCount = columnCount,
+                comment = {
+                    Comment(
+                        comment = it,
+                        onClick = viewModel::upVote
+                    )
+                },
+                commentInput = {
+                    CommentInput(
+                        value = session.commentValue,
+                        onValueChange = viewModel::valueChangedComment,
+                        onSubmit = viewModel::submitComment,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                },
```

Comment:
Should this be hidden if `commentVoteItemId` is null?

Reply:
Good idea!